strings/valentines.json

```
@@ -82,7 +82,7 @@
 		"Your face gets me higher than omega weed.",
 		"Someone check the tachyon doppler array, because you just rocked my world.",
 		"I wish I were your family heirloom so I could be with you all the time.",
-		"How much damage does a toxins bomb do? Enough to break the ice.",
+		"How much damage does a ordnance bomb do? Enough to break the ice.",
```

```suggestion
		"How much damage does an ordnance bomb do? Enough to break the ice.",
```
these just sound wrong as "ordnance" anyways

---

maybe TTV instead?

---

Hm, that could work.